apps/console/src/features/secrets/components/secrets-list.tsx

```diff
@@ -266,6 +310,94 @@ const SecretsList: FC<SecretsListProps> = (props: SecretsListProps): ReactElemen
         ];
     };
 
+    /**
+     * Resolve the relevant placeholder.
+     *
+     * @return {React.ReactElement}
+     */
+    const showPlaceholders = (): ReactElement => {
+        // When the search returns empty.
+        if (searchQuery && filteredSecrets?.length === 0) {
+            return (
+                <EmptyPlaceholder
+                    action={ (
+                        <LinkButton onClick={ onSearchQueryClear }>
+                            { t("console:develop.placeholders.emptySearchResult.action") }
+                        </LinkButton>
+                    ) }
+                    image={ getEmptyPlaceholderIllustrations().emptySearch }
+                    imageSize="tiny"
+                    title={ t("console:develop.placeholders.emptySearchResult.title") }
+                    subtitle={ [
+                        t("console:develop.placeholders.emptySearchResult.subtitles.0", { query: searchQuery }),
+                        t("console:develop.placeholders.emptySearchResult.subtitles.1")
+                    ] }
+                    data-testid={ `${ testId }-empty-search-placeholder` }
+                />
+            );
+        }
+
+        return null;
+    };
+
+    /**
+     * Handles the `onFilter` callback action from the
```

Comment:
Do we need a line break for this comment?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3122/commits/f91e9dcceaefc721392d013e788c15d70e0c4612